lib/omniauth/strategies/openid_connect.rb

```diff
@@ -1,3 +1,4 @@
+# coding:utf-8
```

Comment:
There's no need to add this line. Ruby 1.X is officially dead, and Ruby 2+ already uses UTF as default.

Reply:
OK. Removed.